pkgs/development/python-modules/symbolic/default.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools-rust,
+  rustPlatform,
+  rustc,
+  cargo,
+  milksnake,
+  cffi,
+  nixosTests,
+}:
+
+buildPythonPackage rec {
+  pname = "symbolic";
+  version = "10.2.1"; # glitchtip currently only works with symbolic 10.x
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "getsentry";
+    repo = "symbolic";
+    tag = version;
+    hash = "sha256-3u4MTzaMwryGpFowrAM/MJOmnU8M+Q1/0UtALJib+9A=";
+    # for some reason the `py` directory in the tarball is empty, so we fetch the source via git instead
+    forceFetchGit = true;
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit pname version src;
+    inherit postPatch;
```

Comment:
```suggestion
    inherit pname version src postPatch;
```

Reply:
`nixfmt` would change this to
```nix
    inherit
      pname
      version
      src
      postPatch
      ;
```

Reply:
also fine. having two inherits is kinda pointless and I gave up fighting the formatter